### PR TITLE
Upgrade to ember-cli@2.16.2

### PR DESCRIPTION
**Pull request by tmquinn**

### CODE REVIEW

**Comment by tmquinn** [comment]
Looking into test errors...

File: README.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ export { default } from 'ember-strict-resolver';
 
 # Development of the addon
 
-* `git clone <repository-url>` this repository
-* `cd strict-resolver`
+* `git clone ember-strict-resolver`
```
Comment (rwjblue) [inline README.md]: This needs to be tweaked a bit...

Comment (tmquinn) [reply README.md]: There's an issue for re-writing the README, but are you saying there's something wrong specifically with the change I made? As I don't see it...

Comment (rwjblue) [reply README.md]: That isn’t a valid git clone URL. Should likely be `git@github.com:stefanpenner/ember-strict-resolver.git`

Comment (tmquinn) [reply README.md]: Doh!

**Comment by gabrielcsapo** [comment]
superseded by https://github.com/stefanpenner/ember-strict-resolver/pull/18

**Pull request closed without merging**